config/liota.conf

```diff
@@ -4,13 +4,10 @@ json_path = /etc/liota/logging.json
 [LOG_PATH]
 log_path = /var/log/liota
 
-[UUID_PATH]
-uuid_path = /etc/liota/uuid.ini
-
 [IOTCC_PATH]
-dev_file_path = /etc/liota/conf/devs
-entity_file_path = /etc/liota/conf/entity
-iotcc_path = /etc/liota/conf/iotcc.json
+dev_file_path = /var/tmp/liota/devs
```

Comment:
Let`s try to keep one path either "/usr/lib/liota" or "/var/tmp/liota". Different directories for files emitted by Liota will be difficult to manage at un-installation.

Reply:
use /usr/lib/liota